src/Client/Settings.java

```diff
@@ -567,6 +568,14 @@ public static void definePresets(Properties props) {
     SHOW_BUFFS.put("heavy", true);
     SHOW_BUFFS.put("all", true);
     SHOW_BUFFS.put("custom", getPropBoolean(props, "show_buffs", SHOW_BUFFS.get("default")));
+    
+    SHOW_LAST_MENU_ACTION.put("vanilla", false);
+    SHOW_LAST_MENU_ACTION.put("vanilla_resizable", false);
+    SHOW_LAST_MENU_ACTION.put("lite", false);
+    SHOW_LAST_MENU_ACTION.put("default", true);
```

Comment:
should be default `false`. it's more like debug info

Reply:
Maybe if the names weren't ALL_UPPERCASE_WITH_UNDERSCORES it would look better to be shown to users by default...

Reply:
changed to default false